src/components/contextual/pages/pool/StakingIncentivesCard/StakingIncentivesCard.vue

```diff
@@ -0,0 +1,232 @@
+<script setup lang="ts">
+import useNumbers, { FNumFormats } from '@/composables/useNumbers';
+import { computed, onBeforeMount, ref } from 'vue';
+
+import AnimatePresence from '@/components/animate/AnimatePresence.vue';
+import useStaking from '@/composables/staking/useStaking';
+import { FullPool } from '@/services/balancer/subgraph/types';
+import { bnum } from '@/lib/utils';
+import useTokens from '@/composables/useTokens';
+import { getAddress } from 'ethers/lib/utils';
+import StakePreview from '../../../stake/StakePreview.vue';
+
+type Props = {
+  pool: FullPool;
+};
+const props = defineProps<Props>();
+/**
+ * STATE
+ */const isLoading = ref(false);
+const hasIncentive = true;
+const isStakePreviewVisible = ref(false);
+const stakeAction = ref('');
+const stakedShares = ref('0');
```

Comment:
Receiving `stakedShares` from contract pending response from Fabio regarding whether `balance` from gauge subgraph is cumulative or not.